### DISCUSSION TITLE
After installing mitmproxy-10.2.4-windows-x86_64-installer.exe --script argument not work

### DISCUSSION THREAD

**Discussion by yangfan6888** [mitmproxy/mitmproxy#6791] [Q&A]
On a Windows machine that does not have python installed, download and install mitmproxy-10.2.4-windows-x86_64-installer.exe to run mitmdump.exe, mitmproxy.exe, and mitmweb.exe. But the python script plugin cannot be executed with the --script argument

**Comment by Prinzhorn** [comment]
> But the python script plugin cannot be executed with the --script argument

"cannot be executed" is not a very helpful description
